zerver/openapi/zulip.yaml

```diff
@@ -9018,67 +9026,78 @@ paths:
                 last_update_id:
                   type: integer
                   description: |
-                    Specifies what presence data the client already has, allowing the
-                    server to only return presence data more recent than that.
+                    The identifier that specifies what presence data the client already
+                    has received, which allows the server to only return more recent
+                    user presence data.
 
-                    This parameter should be set to `-1` during initialization of the
-                    client in order to fetch all data, unless the client is obtaining
-                    initial presence metadata from the [`POST /register` endpoint](/api/register-queue).
+                    This should be set to `-1` during initialization of the client in
+                    order to fetch all user presence data, unless the client is obtaining
+                    initial user presence metadata from the
```

Comment:
Just wondering whether there's a good reason we're calling that metadata? I'm guessing it's "metadata" because it refers to fetching the `last_update_id` value - but that happens when fetching presence **data** in general, so might be more readable to keep things simple and not introduce a new term. So maybe just `initial user presence data`?

The rest of the doc looks good